ui/component/viewers/videoViewer/view.jsx

```diff
@@ -27,9 +27,11 @@ import { getAllIds } from 'util/buildHomepage';
 import type { HomepageCat } from 'util/buildHomepage';
 import debounce from 'util/debounce';
 import { formatLbryUrlForWeb, generateListSearchUrlParams } from 'util/url';
+import useInterval from 'effects/use-interval';
 
 // const PLAY_TIMEOUT_ERROR = 'play_timeout_error';
 // const PLAY_TIMEOUT_LIMIT = 2000;
+const PLAY_POSITION_SAVE_INTERVAL = 15000;
```

Comment:
Minor:  maybe rename to `PLAY_POSITION_SAVE_INTERVAL_MS`, although it is obvious.